Android/config/quality/checkstyle_config.xml

```diff
@@ -4,7 +4,6 @@
     "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
 
 <module name="Checker">
-
```

Comment:
Since you are not making any meaningful change to this file, just remove this file from your commits, because it unnecesaarily puts the Git-Blame for this file on you and this PR which shouldn't be the case.
If you're not able to checkout this file from the Git, just add the empty line again to make sure this change is lost